artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/QueueControlImpl.java

```diff
@@ -63,6 +63,7 @@
 public class QueueControlImpl extends AbstractControl implements QueueControl {
 
    public static final int FLUSH_LIMIT = 500;
+   public static final String UNDEFINED = "*_UNDEFINED_*";
```

Comment:
This isn't needed, if you getobjectproperty and do Object.toString on it, the java standard representation "null" will return.